src/components/Table/RowCell.tsx

```diff
@@ -4,12 +4,18 @@ interface RowCellProps {
 	children?: React.ReactNode;
 	className?: string;
 	wrap?: boolean;
+	active?: boolean;
```

Comment:
Per the earlier comment about not needing `active` on a cell, let's revert the changes in this file.